Jenkinsfile

```
@@ -46,6 +46,7 @@ pipeline {
                     }
                     steps {
                         dir("$WORKSPACE/packages/frontend") {
+                            sh 'yarn lint'
                             sh 'yarn install --frozen-lockfile'
```

```suggestion
                            sh 'yarn install --frozen-lockfile'
                            sh 'yarn lint'
```
Might need to swap these round? The Jenkins build is currently failing because it can't find `eslint`